Mathlib/CategoryTheory/Bicategory/Opposite.lean

```diff
@@ -0,0 +1,82 @@
+/-
+Copyright (c) 2025 Christian Merten. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christian Merten, Joël Riou
+-/
+import Mathlib.CategoryTheory.Bicategory.Functor.Pseudofunctor
+
+/-!
+# The opposite of a bicategory
+-/
+
+universe w v u
+
+namespace CategoryTheory
+
+open Opposite
+
+namespace Bicategory.Opposite
+
+variable {B : Type u} [Bicategory.{w, v} B]
+
+instance {X Y : Bᵒᵖ} : Category.{w} (X ⟶ Y) :=
```

Comment:
I know this is still WIP, but could this not cause issues when taking the opposite of a strict bicategory? Then e.g. `Catᵒᵖ` has two category structures, one coming from the (strict) opposite bicategory, and one coming from the opposite underlying category. At some point last year I was also thinking about the opposite bicategory and I thought that one should have another type synonym to talk about this one, but maybe this is not an actual issue? I'm curious because I plan to clean up some old code on 2-Yoneda and in this case I need to talk about the opposite bicategory.

Reply:
It seems to me that both instances are defeq:
```lean
-- TODO: `Strict C` implies `Strict Cᵒᵖ`:
example {C : Type*} [Bicategory C] [Strict C] [Strict Cᵒᵖ] :
    @StrictBicategory.category Cᵒᵖ Bicategory.Opposite.instOpposite _ =
      @Category.opposite C (StrictBicategory.category C) := 
  rfl
```

Reply:
Actually, have I understood correctly in that you are definining the "coop" dual (as in [here](https://math.stackexchange.com/questions/184978/dual-of-a-2-category)). So you are dualizing both the 1-cells and the 2-cells?